### PR TITLE
Recipient API created

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: src/ProjectOrigin.Stamp.Server/Database/Postgres/Scripts/0001.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS Recipients (
```
Comment (TopSwagCode) [inline src/ProjectOrigin.Stamp.Server/Database/Postgres/Scripts/0001.sql:1]: SQL? ?!?!?!?! 😞 (joke)

File: src/ProjectOrigin.Stamp.Server/Database/UnitOfWork.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+        _lazyTransaction.Value.Rollback();
+
+        ResetUnitOfWork();
+    }
+
+
+    public T GetRepository<T>(Func<IDbConnection, T> factory) where T : class
+    {
+        if (_repositories.TryGetValue(typeof(T), out var foundRepository))
```
Comment (TopSwagCode) [inline src/ProjectOrigin.Stamp.Server/Database/UnitOfWork.cs:62]: Why not use built in IOC Container?

File: src/ProjectOrigin.Stamp.Test/Extensions/HttpClientExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public static async Task<T?> ReadJson<T>(this HttpContent content)
+    {
+        var options = GetJsonSerializerOptions();
+        return await content.ReadFromJsonAsync<T>(options);
+    }
+
+    private static JsonSerializerOptions GetJsonSerializerOptions()
+    {
+        var options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
```
Comment (TopSwagCode) [inline src/ProjectOrigin.Stamp.Test/Extensions/HttpClientExtensions.cs:17]: services.AddControllers().AddJsonOptions(options =>
    {
        options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
        options.JsonSerializerOptions.PropertyNamingPolicy=JsonNamingPolicy.CamelCase;
    });

If its only for controller json serializeing.

**Review by TopSwagCode — APPROVED** [review]

File: src/ProjectOrigin.Stamp.Server/Database/Postgres/Scripts/0001.sql (hunk 1 above)
Comment (tnickelsen) [reply src/ProjectOrigin.Stamp.Server/Database/Postgres/Scripts/0001.sql:1]: Yes. I was torn with regards to using dapper and chose to keep using it in Project Origin projects to keep the style the same

File: src/ProjectOrigin.Stamp.Server/Database/UnitOfWork.cs (hunk 2 above)
Comment (tnickelsen) [reply src/ProjectOrigin.Stamp.Server/Database/UnitOfWork.cs]: I have changed this, let me know what you think

**Review by TopSwagCode — APPROVED** [review]

**Review by ckr123 — APPROVED** [review]

**Pull request merged**